test/plugin/test_out_forward.rb

```diff
@@ -107,6 +107,28 @@ def test_configure_no_server
     end
   end
 
+  def test_configure_text_compress
+    d = create_driver
+    assert_equal :text, d.instance.compress
+  end
+
+  def test_configure_gzip_compress
+    d = create_driver(CONFIG + %[compress gzip])
+    assert_equal :gzip, d.instance.compress
+    assert_equal :gzip, d.instance.buffer.compress
+  end
+
+  def test_configure_gzip_compress_in_buffer
```

Comment:
More detailed test description is needed.
And, please add a assertion for log message.